src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -276,6 +301,32 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         }
     }
 
+    /** initialize courses attribute by retreiving recently accessed courses from the server */
+    getRecentlyAccessedCourses() {
+        this.courseService.findAllForDashboard().subscribe({
+            next: (res: HttpResponse<CoursesForDashboardDTO>) => {
+                if (res.body) {
+                    const courses: Course[] = [];
+                    for (const courseDto of res.body.courses) {
+                        courses.push(courseDto.course);
+                    }
+                    this.courses = courses.sort((a, b) => (a.title ?? '').localeCompare(b.title ?? ''));
```

Comment:
In this case it might be a matter of taste - I think it is better to read if the variable name is a bit more descriptive
```suggestion
                    this.courses = courses.sort((courseA, courseB) => (courseA.title ?? '').localeCompare(courseB.title ?? ''));
```

We also seem to have the exact line of code in `courses.component.ts` - I think it would be better to introduce a shared method to ensure that the sorting will stay the same (e.g. if a bug is found in the future and it might only be fixed in one component otherwise).

Reply:
agreed